crates/floresta-cli/Cargo.toml

```diff
@@ -20,11 +20,7 @@ bitcoin = { version = "0.32", features = ["serde", "std"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 anyhow = "1.0"
-jsonrpc = { version = "0.18.0", optional=true, features = ["minreq_http"] }
-
-[features]
-default = ["with-jsonrpc"]
-with-jsonrpc = ["jsonrpc"]
+jsonrpc = { version = "0.18.0", features = ["minreq_http"] }
```

Comment:
The feature-gate here is useful if you're using `floresta-chain` as a library, rather than a binary. I still think it's useful, since you can pick the `http` server you prefer

Reply:
Well seen!